docs/team/davelowqx.md

```diff
@@ -1,52 +1,29 @@
 ---
 layout: page
-title: Dave Low's Project Portfolio Page
+title: Low Qi Xiang's Project Portfolio Page
 ---
 
 ### Project: AddressBook Level 3
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Managera is a **desktop app that provides event organisers with a convenient method of keeping track of upcoming events and the details of their participants.**
+It does not handle communication between the event organiser and the participants. This project is adapted from SE-EDU AddressBook - Level 3.
 
 Given below are my contributions to the project.
 
-- **New Feature**: Added the ability to undo/redo previous commands.
+- **New Feature**: Adding Events
 
-  - What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  - Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  - Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  - Credits: _{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}_
+  - What it does: Adds an Event to Managera.
```

Comment:
You can write a lot more than this for the new feature, don't sell yourself short! 😄 
It is a core feature so there should be plenty that you can write about it. If you want you can look at some of the PPPs that the rest of us have written to get an idea. You can elaborate more of how a user would go about using this feature and why they would need it.

Reply:
okok thanks bro will see what I can add on 😅😅😅